simplenote_cli/sncli.py

```diff
@@ -1081,6 +1081,22 @@ def cli_note_create(self, from_stdin, title):
             self.ndb.create_note(content)
             self.sync_notes()
 
+    def cli_note_import(self, from_stdin):
+
+        if from_stdin:
+            raw = ''.join(sys.stdin)
+        else:
+            raw = self.exec_cmd_on_note(None)
```

Comment:
Is this then using an editor to enter the raw json note? Wouldn't it be better to specify either the file or json string on the command line?